programmers/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,22 @@
+function solution(lottos, win_nums) {
+    let answer = [];
+    let zeroCnt = 0;
+    let Cnt = 0;
+    let rank = [6,6,5,4,3,2,1];
+    
+    let numSet = new Set();
+    for (let i = 0; i < win_nums.length; i++) {
+        numSet.add(win_nums[i])
+    }
+    for (let i = 0; i < lottos.length; i++) {
+        if (numSet.has(lottos[i])) { 
+            Cnt++;
+        }
+        else if (lottos[i]==0) {
+            zeroCnt++;
+        }
+    }
```

Comment:
1. if...else 문 한 줄로 할 수 있으면 한줄로 표현하기
2. ==(동등 연산자)말고 ===(일치 연산자) 사용하기
```suggestion
    for (let i = 0; i < lottos.length; i++) {
        if (numSet.has(lottos[i])) cnt++;
        else if (lottos[i] === 0) zeroCnt++;
    }
```